### ISSUE TITLE
The plug-in gets stuck on the loading screen

### ISSUE THREAD

**Issue by PatAbah** [7HR4IZ3/acode-plugin-workspace#1]
Hello there!
I found your plug-in on Acode and decided to give it a try since it's a nice concept. However, after installing, clicking any of the two buttons on the side menu just gives a "Loading" modal that lasts forever as seen in the picture I attached. 

I haven't been able to use the plug-in.
I'm on v1.8.7 (330). What could be the cause of this? 
![Screenshot_20240104-164818](https://github.com/7HR4IZ3/acode-plugin-workspace/assets/34959120/8bae03a4-f98e-4937-ab44-9c52aef51aa1)

**Comment by 7HR4IZ3** [comment]
Can you share a screenshot of the console when the loading dialog is showing.. Use Acode SDK plugin to see the console..

**Comment by PatAbah** [comment]
> Can you share a screenshot of the console when the loading dialog is showing.. Use Acode SDK plugin to see the console..

Thanks for the quick response. Also thanks for the suggested plug-in! Very useful for debugging. I realized I could now fix certain errors I encounter with plug-ins myself 😊

Well for the error I get, it says `Cannot read property 'writeFile' of undefined... Line 134...main.js` when I click "Save Workspace" 

Then error on line 15: can't read property `readFile` of undefined as well.

**Comment by 7HR4IZ3** [comment]
Okay, I know what's wrong..

When you click on Save Workspace, where do you select the file to save to from?

The plugin only saves to files in the internal storage, termux storage is not currently supported and you have to save the file first..

If that's not the problem then update your app, latest version is 1.9.0

**Comment by 7HR4IZ3** [comment]
Join the acode telegram page for faster response and help from other plugin developers plus new app version and plugins are released there..

**Comment by PatAbah** [comment]
Didn't know there was a Telegram group. Thanks!

**Comment by PatAbah** [comment]
> When you click on Save Workspace, where do you select the file to save to from?

I don't see any modal to Select Save Location when I click the buttons...it just starts loading immediately. There's no option to select file. 

> If that's not the problem then update your app, latest version is 1.9.0

Well, the latest version is 1.8.7 (330) on Google play. I looked for Acode on another repo...and I saw v1.9.0 (325) which is kinda weired since 325 is a downgrade from 330. 
I tried it out just to see if your plug-in would work, but still the same error. 

> The plugin only saves to files in the internal storage, termux storage is not currently supported and you have to save the file first..

I closed all files from Termux in the editor, still nothing.
